### DISCUSSION TITLE
[ClassNotFound error] After installing SQL Lineage Support in Kyuubi=1.9.3 according to the Doc, ClassNotFound error appears in thread spark-listener-group-shared

### DISCUSSION THREAD

**Discussion by madeirak** [apache/kyuubi#6800] [Q&A]
spark: 3.3.1
Kyuubi: 1.9.3
branch: branch-1.9

ref: https://kyuubi.readthedocs.io/en/v1.9.2/extensions/engines/spark/lineage.html
Packaging spark lineage jar with `build/mvn clean package -pl :kyuubi-spark-lineage_2.12 -am -DskipTests -Dspark.version=3.3.1` as show in Doc above.

I am using the `branch-1.9` branch of kyuubi, the pom.xml is 
![image](https://github.com/user-attachments/assets/53a8f829-8f4e-4132-98c9-e875d2f8ce2b)
"provide", the jar package will not contain kyuubi-evetns_2.12.xx dependency...
But when I complete according to the process of Doc, I got this error when running Spark sql, I wonder how to deal with it 
![img_v3_02ge_af84a9cf-69da-4fcf-83f0-8719d685768g](https://github.com/user-attachments/assets/6c0ae064-3c42-456b-921c-f369bc9c50c0)

**Comment by pan3793** [comment]
cc @wForget

**Reply by wForget** [reply]
Looks like a duplicate of https://github.com/apache/kyuubi/issues/6703, I'll try to fix it.

**Reply by madeirak** [reply]
> Looks like a duplicate of #6703, I'll try to fix it.

Thx for it , will fix code be submitted to branch `branch-1.9`?
